src/main/java/ch/uzh/ifi/hase/soprafs23/entity/User.java

```diff
@@ -39,21 +42,19 @@ public class User implements Serializable {
     private UserStatus status;
 
     @Column(nullable = true)
-    private String allergies;
+    @ElementCollection
+    private Set<String> allergies = new HashSet<>(); // important to initialize as an empty set
```

Comment:
I think it's a good point with the making clear that it is a set, maybe then renaming the field to allergiesSet here? following the naming scheme of the DTOs as well

Reply:
alright! adjusted